File: FizzBuzz/fizz_buzz.py


def fizzBuzz(arr: list) -> list:
    for num in arr:
        if num % 3 ==0 and num % 5 == 0:
            print(f"{num}: FizzBuzz")
        elif num % 3 == 0:
            print(f"{num}: Fizz")
        elif num % 5 == 0:
            print(f"{num}: Buzz")

def main():
    fizzBuzz(list(range(1,21)))

if __name__ == '__main__':
    main()

File: FizzBuzz/FizzBuzz.scala
object FizzBuzz extends App {
  if (args.length != 1) {
    println("Error !!! Provide input number ")
    System.exit(0)
  }
  for (num <- 1 to args(0).toInt) {
    if (num % 15 == 0)
      println("FizzBuzz")
    else if (num % 3 == 0)
      println("Fizz")
    else if (num % 5 == 0)
      println("Buzz")
    else
      println(num)
  }
}
